apps/jobs/views.py

```diff
@@ -235,6 +239,7 @@ def change_submission_data_and_visibility(request, challenge_pk, challenge_phase
                                       context={
                                                'participant_team': participant_team,
                                                'challenge_phase': challenge_phase,
+                                               'challenge_id': challenge_pk,
```

Comment:
Why do we need this?

Reply:
This is used by `to_representation` in `SubmissionSerializer`.